docker-for-mac/troubleshoot.md

```diff
@@ -352,3 +353,91 @@ in the Apple documentation, and Docker Desktop [Mac system requirements](install
   sudo launchctl unload /System/Library/LaunchDaemons/org.ntp.ntpd.plist
   sudo launchctl load /System/Library/LaunchDaemons/org.ntp.ntpd.plist
   ```
+
+## Support
+
+This section contains information on how to get support on Docker Desktop.
+
+>**Note**
+>
+> Docker Desktop offers support for Pro and Team users only. For more information about the pricing plans, see [Docker Pricing](https://www.docker.com/pricing){: target="_blank" rel="noopener" class="_"}.
+
+### How do I get Docker Desktop support?
+
+If you have subscribed to a Pro and Team account, please raise a ticket through [Docker Desktop support](https://hub.docker.com/support/desktop/){:target="_blank" rel="noopener" class="_"}.
+
+Docker Community users can get support through our Github repos for-win and for-mac, where we respond on a best-effort basis.
+
+### What support can I get?
+
+If you are a Pro or a Team user, you can request for support on the following types of issues:
+
+* Desktop upgrade issues
+* Desktop installation issues
+  * Enabling virtualization in BIOS
+  * Enabling Windows features
+  * Genuine installation crashes
+  * Crashes due to conflicts with other software
+  * Installation issues from enterprise deployment
+* First run issues
+   * Failure to launch Docker Desktop on first run
+* Usage issues
+  * Crash closing software
+  * Docker Desktop Not behaving as expected
+* Configuration issues
+* Basic product ‘how to’ questions such as ‘how to work efficiently on WSL 2’
+
+### What is not supported?
+
+Docker Desktop excludes support for the following types of issues:
+
+* Use on or in conjunction with hardware or software other than that specified in the applicable documentation
+* Running on unsupported operating systems, including beta/preview versions of operating systems
+* Use of Docker/support for learning or fixing Docker commands
+* Support for the Docker engine, Docker CLI, or other bundled Linux components
+* Support for Kubernetes
+* Features labeled as experimental
+* System/Server administration activities
+* Supporting Desktop as a production runtime
+* Scale deployment/multi-machine installation of Desktop
+* Routine product maintenance (data backup, cleaning disk space and configuring log rotation)
+* Third-party applications not provided by Docker
+* Altered or modified Docker Software
+* Defects in the Docker Software due to hardware malfunction, abuse or improper use
+* Any version of the Docker software other than the latest version
+* Reimbursing and expenses spent for third-party services not provided by Docker
+* Docker Support excludes training, customization and integration
+
+### What versions are supported?
+
+We currently only offer support for the latest version of Docker Desktop. If you are running an older version, you may be asked to upgrade before reviewing your support request.
+
+### Can I deploy Docker Desktop to my team?
```

Comment:
Let's leave this section out for the moment, until we can check with @nebuk89. I'm not sure we've made the necessary changes to the EULA, and I don't want to get ahead of that.